lib/base-chips/field-types/file.js

```diff
@@ -3,6 +3,8 @@ const locreq = require("locreq")(__dirname);
 const Promise = require("bluebird");
 const File = locreq("lib/data-structures/file.js");
 const FileManager = locreq("lib/core-services/file-manager.js");
+const rp = require("request-promise");
```

Comment:
I can see that you're using `request-promise`. :) It's far better than the regular `request`.